pylib/cqlshlib/cql3handling.py

```diff
@@ -517,6 +518,8 @@ def cf_prop_val_completer(ctxt, cass):
         return [Hint('<\'none\'|\'blocking\'>')]
     if this_opt == 'allow_auto_snapshot':
         return [Hint('<boolean>')]
+    if this_opt == 'auto_snapshot_ttl':
+        return [Hint('<duration>')]
```

Comment:
duration is seconds, minutes, hours or days. (s, m, h, d). This nicely relates to what DurationSpec is based on. Duration spec has resolution to `ns` but anything less than seconds for snapshot ttl is hardly applicable.